app/assets/stylesheets/bourbons.scss

```diff
@@ -57,3 +70,83 @@ $black: #000;
   border: 3px solid $gold;
   color: $black;
 }
+
+//SHOW PAGE
+//==============================
+
+.bourbon-pic {
+  background-color: $black;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
```

Comment:
Properties should be ordered background-color, background-position, background-repeat, background-size, display, flex-direction, height, justify-content, width